src/mscorlib/src/System/Diagnostics/Eventing/EventSource.cs

```diff
@@ -5368,19 +5322,6 @@ public static void DisableFilter(ref ActivityFilter filterList, EventSource sour
         }
 
         /// <summary>
-        /// Returns the first ActivityFilter from 'filterList' corresponding to 'source'.
-        /// </summary>
-        public static ActivityFilter GetFilter(ActivityFilter filterList, EventSource source)
-        {
-            for (var af = filterList; af != null; af = af.m_next)
-            {
-                if (af.m_providerGuid == source.Guid && af.m_samplingFreq != -1)
-                    return af;
-            }
-            return null;
-        }
-
-        /// <summary>
         /// Returns a session mask representing all sessions in which the activity 
         /// associated with the current thread is allowed  through the activity filter. 
         /// If 'triggeringEvent' is true the event MAY be a triggering event. Ideally 
```

Comment:
Should I introduce an #if CORECLR for these? Eg WriteEvent(..) are quite large. Don't know how much we care about size.

---
In reply to: [99062226](https://github.com/dotnet/coreclr/pull/9269#discussion_r99062226) [](ancestors = 99062226)